low-level/json_msgs/messages/sensors/cpu_data.py

```diff
@@ -88,6 +89,10 @@ def __init__(self, host_id,
         self._node_id           = node_id
         self._cluster_id        = cluster_id
         self.alert_type         = alert_type
+        self.event              = event
+
+        if self.alert_type == "fault_resolved":

```

Comment:
hey sorry, why we are deciding alert type in messages ? It should be coming from sensor itself ... !

Reply:
Yes, alert_type is coming from senor itself, But for these node OS level alerts , previously we  had defined severity here as "warning" for all types of alert, 
So I just add check here that if alert_type is "fault_resolved", we are making severity as "informational".

Reply:
Ok i meant for severity, which is getting defined here only. So to keep changes minimal for SMS2, sure we will update here it self.

Reply:
i meant for severity, which is getting defined during json formation only. Fine, so to keep changes minimal for SMS2, sure we will update here it self.